src/lib/elementary/elm_main.c

```diff
@@ -2,17 +2,14 @@
 # include "elementary_config.h"
 #endif
 
-#ifndef _WIN32
-# include <dlfcn.h> /* dlopen,dlclose,etc */
-#endif
-
 #ifdef HAVE_CRT_EXTERNS_H
 # include <crt_externs.h>
 #endif
 
 #ifdef _WIN32
-# include <direct.h> /* getcwd */
 # include <evil_private.h> /* dlopen,dlclose,etc */
+# include <dlfcn.h> /* dlopen,dlclose,etc */
```

Comment:
This should go to the `#else` side, as it should be included only when not on windows.